ops/testing.py

```diff
@@ -708,6 +728,9 @@ def __init__(self, unit_name, meta):
         self._unit_status = {'status': 'maintenance', 'message': ''}
         self._workload_version = None
         self._resource_dir = None
+        # {socket_path : _TestingPebbleClient}
```

Comment:
Could say this with an actual `# type: T` comment on the assignment line?

Reply:
so I added one, the only problem is that 'str' doesn't tell you what that str is like 'socket_path' tells you the meaning of the key.
So for now I have both, but if you'd prefer a different way of documenting it, I'm happy to change it.